Clover/app/src/main/java/org/floens/chan/ui/view/MultiImageView.java

```diff
@@ -180,6 +186,16 @@ public CustomScaleImageView findScaleImageView() {
         return bigImage;
     }
 
+    public GifImageView findGifImageView() {
+        GifImageView gif = null;
+        for (int i = 0; i < getChildCount(); i++) {
+            if (getChildAt(i) instanceof GifImageView) {
+                gif = (GifImageView) getChildAt(i);
```

Comment:
do we want to `break` or just return here to stop iterating if we find one? or do we want to return the last one found?

Reply:
The same code essentially exists for `findImageScaleView`, so it's getting the last one. Although I think that there should only ever be one instance (the "center" image in the pager), there's no harm in leaving it the way it is, the number of children is super small.